manifests/install.pp

```diff
@@ -15,14 +15,27 @@
   }
   #TODO Configure Postfix properly, maybe as satellites and have a mail server to handle all our sent mail?
 
-~>file { '/etc/network/interfaces':
-    ensure  => 'present',
-    content => template('proxmox/interfaces.erb'),
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644'
+  network::interface { 'vmbr0':
+    family       => 'inet',
+    ipaddress    => $::ipaddress,
+    netmask      => $::netmask,
+    gateway      => $gatewayv4,
+    bridge_ports => [ $::netdev ],
```

Comment:
This breaks CI:
manifests/install.pp - WARNING: top-scope variable being used without an explicit namespace on line 22